specs/csp-pinning/index.src.html

```diff
@@ -97,10 +97,14 @@ <h3 id="use-cases">Use Cases</h3>
                                style-src https://application2.cdn.com;
     </pre>
 
+		ISSUE: This example contradicts advice elsewhere that the
+    value of the header SHOULD be the same for every <a>resource
+    representation</a>.
```

Comment:
Looking again, I don't understand this issue. The pinned header _is_ the same in both examples, isn't it?